CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contribution Guide
+
+## Welcome Section
+- **Introduction**: Welcome dear contributors to the Tegel Design system project. We are very thankful for your interest in contributing.
+
+- **Project Overview**: The aim of Tegel Design system is to provide a set of reusable components that can be used in any project. The project is open source and we welcome contributions from the community.
+
+## How to Contribute
+- **Types of Contributions**: There are different ways to contribute to the project. You can contribute by writing code, documentation, reporting bugs, requesting new features, or providing community support in our [Development support channel](https://teams.microsoft.com/l/channel/19%3a5e33f67fe502441f914fbcdc6e2548f5%40thread.skype/Development%2520support%2520-%2520Tegel?groupId=79f9bfeb-73e2-424d-9477-b236191ece5e&tenantId=3bc062e4-ac9d-4c17-b4dd-3aad637ff1ac) or use [GitHub Discussions](https://github.com/scania-digital-design-system/tegel/discussions) to start new topic.
+- **First-Time Contributors**: We recommend taking a look on the reported [issues](https://github.com/scania-digital-design-system/tegel/issues) here on GitHub. We consider a small bug or improvement a best candidate for first contribution. 
+
+## Before You Start
+- **Type of Contribution**: If the contribution is a new component or a significant change to an existing one, please get in touch with Tegel's designers and developers beforehand to align on the direction.
+- **Check Existing Issues/PRs**: Take a look on the [issues](https://github.com/scania-digital-design-system/tegel/issues) and [pull requests](https://github.com/scania-digital-design-system/tegel/pulls) to see if someone has already started on the same thing.
+
+## Contribution Process
+- **Repo access**: You can either fork the repo or in case you are Scania employee you can ask for access to the repo as external contributor.
```

Comment:
```suggestion
- **Repo access**: You can either fork the repo or in case you are a Scania employee you can ask for access to the repo as an external contributor.
```